src/modules/Home/components/Home/Home.tsx

```diff
@@ -0,0 +1,324 @@
+import cn from 'classnames';
+import React from 'react';
+
+import boxSVG from 'assets/images/box.svg';
+import ChipIcon from 'assets/images/ChipIcon';
+import worldSVG from 'assets/images/continents.svg';
+import devicesSVG from 'assets/images/devices.svg';
+import discountSVG from 'assets/images/discount.svg';
+import githubSVG from 'assets/images/github.svg';
+import LogoIcon from 'assets/images/LogoIcon';
+import octocatSVG from 'assets/images/octocat.svg';
+import openedBookSVG from 'assets/images/openedBook.svg';
+import shieldSVG from 'assets/images/shield.svg';
+import userSVG from 'assets/images/user.svg';
+import Flex from 'common/components/Flex/Flex';
+import MainSlider from 'modules/Home/containers/MainSliderContainer';
+
+import styles from './home.scss';
+
+interface IProps {
+	currentMainSlide: any;
+}
+
+const Home: React.FC<IProps> = props => (
+	<div className={cn(styles.home, styles[props.currentMainSlide.name])}>
+		<Flex justifyContent='center' className={styles.intro}>
+
+			<Flex alignItems='center' direction='column' className={styles.mainTitles}>
+				<LogoIcon fill='#FFFFFF' className={styles.introLogo} />
+				<p className={styles.mainTitles_top}>Introducing CusDeb</p>
+				<p className={styles.mainTitles_bottom}>
+					CusDeb is a powerful service for building customized operating system
+					images for single-board computers
+				</p>
+			</Flex>
+
+		</Flex>
+
+		<MainSlider slideTiming={100000} />
+
+		<div className={styles.indicators}>
+			<Flex className={styles.homeContainer}>
+				<Flex className={styles.indicator} alignItems='center'>
+					<Flex
+						justifyContent='center'
+						alignItems='center'
+						className={styles.indicatorCircle}
+					>
+						<img className={styles.indicatorCircleImg} src={boxSVG} alt=''/>
```

Comment:
why you use empty alt?

Reply:
![image](https://user-images.githubusercontent.com/9141710/55798822-8b6b1f80-5ad8-11e9-8e32-1b37e894bf91.png)

HTML validator

Reply:
And chrome audit also barking on those alt's